doc/conf.py

```diff
@@ -95,8 +93,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This patterns also effect to html_static_path and html_extra_path
-exclude_patterns = ['auto_examples/index.rst', '_build', 'Thumbs.db',
-                    '.DS_Store', "**.ipynb_checkpoints", 'auto_examples/*.rst']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', "**.ipynb_checkpoints"]
```

Comment:
This was the real problem @adam2392 , for some reason all examples were excluded. But I fixed a bunch of other minor organizational issues while I was in there. It's not perfect but it's good progress and hopefully allows you to start working on docs deployment since in theory now CircleCI should bulid properly